mlflow/langchain/__init__.py

```diff
@@ -113,9 +113,15 @@ def get_default_pip_requirements():
         Calls to :func:`save_model()` and :func:`log_model()` produce a pip environment
         that, at a minimum, contains these requirements.
     """
+    import langchain
+
     # pin pydantic and cloudpickle version as they are used in langchain
     # model saving and loading
-    return list(map(_get_pinned_requirement, ["langchain", "pydantic", "cloudpickle"]))
+    pip_reqs = list(map(_get_pinned_requirement, ["langchain", "pydantic", "cloudpickle"]))
+    # temp fix for https://github.com/langchain-ai/langchain/issues/24287
```

Comment:
can we wait for this issue to be fixed?

Reply:
langchain 2.8.0 is already released and contains this bug 🤔 We can remove this after 2.8.1/2.9.0 is released?